v4l2/v4l2_encode.cpp

```diff
@@ -58,6 +58,15 @@ bool V4l2Encoder::start()
     streamFormat.streamFormat = AVC_STREAM_FORMAT_ANNEXB;
     m_encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 
+    if (m_svct) {
+        VideoTemporalLayers layers;
+        layers.numLayersMinus1 = 1; // It is two layers actually
+        layers.bitRate[0] = m_videoParams.rcParams.bitRate / 2;
+        layers.bitRate[1] = m_videoParams.rcParams.bitRate;
```

Comment:
layers.bitRate[1] = m_videoParams.rcParams.bitRate;
this assignment make the code more clear and it is not conflict with that in libyami either, so suggest to keep this line.